lsp-metals.el

```diff
@@ -163,7 +182,9 @@ Should be ignored if there is no open doctor window."
  (make-lsp-client :new-connection (lsp-stdio-connection 'lsp-metals--server-command)
                   :major-modes '(scala-mode)
                   :priority -1
+                  :custom-capabilities '((experimental (decorationProvider . t)))
```

Comment:
It seems it'll conflict with https://github.com/emacs-lsp/lsp-treemacs/blob/d097670f1d77e0c806b0094df4ebb575fb83ed87/lsp-metals-treeview.el#L811-L822

Reply:
Yes. lsp-treemacs must be changed to append in :experimental.

Reply:
Is this working now ? Do i have to configure something ???